### ISSUE TITLE
There is no way to pass some g-recaptcha tag attributes / grecaptcha.render parameters

### ISSUE THREAD

**Issue by silverdr** [samueljseay/recaptcha#65]
There seem to be no way to pass some important params like e. g. `expired-callback` or `error-callback`.

**Comment by samueljseay** [comment]
@silverdr you're absolutely right that its light on config at the moment and does not support these. I'd welcome a PR, but in the meantime I'm going to add this to a backlog of things that need to be done.
